egs/madcat_zh/v1/local/chain/run_flatstart_cnn1a.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 # Copyright    2017  Hossein Hadian
 
-# local/chain/compare_wer.sh exp/chain/e2e_cnn_1afix1
-# System                      e2e_cnn_1afix1
-# WER                             10.55
-# Final train prob              -0.0702
-# Final valid prob              -0.0578
-# Final train prob (xent)
-# Final valid prob (xent)
+# local/chain/compare_wer.sh exp/chain/cnn_1a/ exp/chain/cnn_chainali_1b/ exp/chain/e2e_cnn_1afix1/
```

Comment:
Please change the affix to 1a (instead of `1afix1`). 
Also, please make sure that the download dir (i.e. the database directory) can be easily set in one location (e.g. in run.sh). Recently, there was an issue related to this for MADCAT Arabic.
@danpovey, except for the two notes above, I think this PR is ready.